src/test/integration/incrementalFeed.spec.ts

```diff
@@ -1,106 +1,501 @@
 import assert from "assert";
-import { FeedOptions } from "../..";
-import { Container } from "../../client";
+import { RequestOptions } from "../..";
+import { Container, ContainerDefinition } from "../../client";
+import { Constants, Helper } from "../../common";
+import { IHeaders } from "../../queryExecutionContext/IHeaders";
 import { getTestContainer, removeAllDatabases } from "../common/TestHelpers";
 
-describe("NodeJS Incremental Feed Tests using 'a_im' and 'IfNoneMatch' options", function() {
-  // delete all databases and create sample database
-  before(async function() {
-    await removeAllDatabases();
-  });
+function hasDupeKey(items: any[]) {
+  if (items && items.length === 0) {
+    return false;
+  }
+  const key = items[0].key;
+  let hasDupe = false;
+  for (const item of items) {
+    if (item.key !== key) {
+      hasDupe = true;
+      break;
+    }
+  }
+  return hasDupe;
+}
 
-  describe("Newly updated documents should be fetched incremetally", function() {
-    let container: Container;
+describe("Change Feed Iterator", function() {
+  this.timeout(process.env.MOCHA_TIMEOUT || 20000);
 
-    // create container and two documents
+  describe("Non-partitioned", function() {
+    // delete all databases and create sample database
     before(async function() {
-      container = await getTestContainer("Newly updated documents should be fetched incrementally");
-      await container.items.create({ id: "doc1" });
-      await container.items.create({ id: "doc2" });
+      await removeAllDatabases();
+    });
+
+    describe("Should only find items after start time", function() {
+      let container: Container;
+
+      // create container and two items
+      before(async function() {
+        container = await getTestContainer("Newly updated items should be fetched incrementally");
+      });
+
+      after(async function() {
+        await container.delete();
+      });
+
+      // TODO: start time is currently being ignored. Need to investigate.
```

Comment:
Is this closed now or still something is not working? If cloesed, let's remove the comment.

Reply:
it's working now, yeah. Good catch!